src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -157,11 +157,8 @@ public String getDescription() {
     }
 
     public String getDescriptionAndLink() {
-        if (StringUtil.isBlank(linkedFile.getDescription())) {
-            return linkedFile.getLink();
-        } else {
-            return linkedFile.getDescription() + " (" + linkedFile.getLink() + ")";
-        }
+        return (StringUtil.isBlank(linkedFile.getDescription()) ? "" : linkedFile.getDescription() + " ")
+                + "(" + linkedFile.getLink() + ")";
```

Comment:
This is not JabRef style. You can rewrite the `if` block if you want.

But I think, the code is OK enough to keep as is.